github/AuthenticatedUser.py

```diff
@@ -498,6 +498,47 @@ def create_fork(self, repo):
             self._requester, headers, data, completed=True
         )
 
+    def create_repo_from_template(
+            self,
+            name,
+            repo,
+            description=github.GithubObject.NotSet,
+            private=github.GithubObject.NotSet,
+    ):
+        """
+        :calls: `POST  /repos/:template_owner/:template_repo/generate <https://developer.github.com/v3/repos/#create-repository-using-a-repository-template>`
+        :param name: string
+        :param repo :class:`github.Repository.Repository`
+        :param description: string
+        :param private: bool
+        :rtype: :class:`github.Repository.Repository`
+        """
+        assert isinstance(repo, github.Repository.Repository), repo
```

Comment:
Should this be:
```suggestion
        assert isinstance(name, str), name
```

Reply:
My first implementation used strings but I changed it to `Repository` to match the pattern in `create_fork`, which is a very similar method.

Reply:
ah well this line looks like a duplicate for the line beneath it

Reply:
Thanks for catching this!